core/include/ldml/keyboardprocessor_ldml.h

```diff
@@ -93,10 +93,10 @@
 #define LDML_LENGTH_VARS_ITEM 0x10
 #define LDML_LENGTH_VKEY 0xC
 #define LDML_LENGTH_VKEY_ITEM 0x8
-#define LDML_MARKER_ANY_INDEX 0xFFFE
+#define LDML_MARKER_ANY_INDEX 0xD7FF
 #define LDML_MARKER_CODE 0x8
-#define LDML_MARKER_MAX_COUNT 0xFFFC
-#define LDML_MARKER_MAX_INDEX 0xFFFD
+#define LDML_MARKER_MAX_COUNT 0xD7FD
+#define LDML_MARKER_MAX_INDEX 0xD7FE
 #define LDML_MARKER_MIN_INDEX 0x1
```

Comment:
Is this possible?

```suggestion
#define LDML_MARKER_MAX_INDEX 0xD7FE
#define LDML_MARKER_MIN_INDEX 0x1
#define LDML_MARKER_MAX_COUNT (LDML_MAX_INDEX - LDML_MIN_INDEX)
```

A brief web search suggests that the C preprocessor can perform integer arithmetic, so I _think_ this should be possible?  This would more closely match line 629 from `keyboardprocessor_ldml.ts` as well.

... though, shouldn't there actually be a final +1 to the value?  I assume that "max index" is still a valid index?

Reply:
The C preprocessor is essentially token replacement, so arithmetic is fine -- it'll be done by the compiler. The parentheses are essential.

```suggestion
#define LDML_MARKER_MAX_INDEX 0xD7FE
#define LDML_MARKER_MIN_INDEX 0x0001
#define LDML_MARKER_MAX_COUNT (LDML_MAX_INDEX - LDML_MIN_INDEX + 1)
```

Reply:
> Note, this file is autogenerated from the .ts (see lines 9-11):

That explanation makes sense.  Those lines don't show up in the sections highlighted for review, so I wasn't aware.